testutils/src/main/scala/za/co/absa/enceladus/testutils/datasetComparison/ComparisonJob.scala

```diff
@@ -21,6 +21,7 @@ import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
 import za.co.absa.enceladus.testutils.exceptions._
 import za.co.absa.enceladus.testutils.{DataframeReader, DataframeReaderOptions, HelperFunctions}
+import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
 
 object ComparisonJob {
   private val log: Logger = LogManager.getLogger("enceladus.testutils.ComparisonJob")
```

Comment:
Would you mind doing the update even to this? 
`private val log: Logger = LogManager.getLogger(this.getClass)`